crates/runtime/src/request/context.rs

```diff
@@ -182,6 +182,12 @@ impl RequestContext {
             extensions.insert(TypeId::of::<T>(), Arc::new(extension));
         }
     }
+
+    pub async fn load_extensions(&self) {
+        if let Some(extension) = self.extension::<DatabricksAuthExtension>() {
```

Comment:
This will need to be reworked to have a proper trait based approach, the request context module shouldn't know anything about databricks. But that can come in a separate PR